packages/cli-helpers/src/lib/project.ts

```diff
@@ -33,21 +36,113 @@ export const getInstalledRedwoodVersion = () => {
   }
 }
 
-export const addEnvVarTask = (name: string, value: string, comment: string) => {
+/**
+ * Updates the project's redwood.toml file to include the specified packages plugin
+ *
+ * Uses toml parsing to determine if the plugin is already included in the file and
+ * only adds it if it is not.
+ *
+ * Writes the updated config to the file system by appending strings, not stringify-ing the toml.
+ */
+export const updateTomlConfig = (packageName: string) => {
+  const redwoodTomlPath = getConfigPath()
+
+  const configLines = []
+
+  const configContent = fs.readFileSync(redwoodTomlPath, 'utf-8')
+  const config = toml.parse(configContent)
+
+  if (!config || !config.experimental || !config.experimental.cli) {
+    configLines.push('[experimental.cli]')
+  }
+
+  if (!config?.experimental?.cli?.autoInstall) {
+    configLines.push('  autoInstall = true')
+  }
+
+  if (!config?.experimental?.cli?.plugins) {
+    // If plugins array is missing, create it.
+    configLines.push('  [[experimental.cli.plugins]]')
+  }
+
+  // Check if the package is not already in the plugins array
+  if (
+    !config?.experimental?.cli?.plugins?.some(
+      (plugin: any) => plugin.package === packageName
+    )
+  ) {
+    configLines.push(`    package = "${packageName}"`)
+  }
+
+  const newConfig = configContent + configLines.join('\n')
+
+  return fs.writeFileSync(redwoodTomlPath, newConfig, 'utf-8')
+}
+
+export const updateTomlConfigTask = (packageName: string) => {
+  return {
+    title: `Updating redwood.toml to configure ${packageName} ...`,
+    task: () => {
+      updateTomlConfig(packageName)
+    },
+  }
+}
+
+export const addEnvVarTask = (
+  name: string,
+  value: string,
+  comment: string,
+  overwrite = false
+) => {
   return {
     title: `Adding ${name} var to .env...`,
     task: () => {
-      const envPath = path.join(getPaths().base, '.env')
-      const content = [comment && `# ${comment}`, `${name}=${value}`, ''].flat()
-      let envFile = ''
+      addEnvVar(name, value, comment, overwrite)
+    },
+  }
+}
 
-      if (fs.existsSync(envPath)) {
-        envFile = fs.readFileSync(envPath).toString() + '\n'
-      }
+export const addEnvVar = (
+  name: string,
+  value: string,
+  comment: string,
+  overwrite = false
+) => {
+  const envPath = path.join(getPaths().base, '.env')
+  const content = [comment && `# ${comment}`, `${name}=${value}`, ''].flat()
+  let envFile = ''
 
-      fs.writeFileSync(envPath, envFile + content.join('\n'))
-    },
+  if (fs.existsSync(envPath)) {
+    envFile = fs.readFileSync(envPath).toString()
+    const lines = envFile.split('\n')
+
+    // Check if the variable already exists
+    const existingIndex = lines.findIndex((line) => {
+      const trimmedLine = line.trim()
+      return (
+        trimmedLine.startsWith(`${name}=`) ||
+        trimmedLine.startsWith(`#${name}=`)
```

Comment:
I personally tend to add a space so it would be `# NAME=VALUE` in my .env but I wouldn't force that we check this too. I just wanted to mention. We could do a regex that would allow any whitespace between the `#` and `name`?

Reply:
Have added better logic to handle existing enviers and spacing